test_requests.py

```
@@ -0,0 +1,14 @@
+
+url = "https://ez-runner.com/rise/members/login"
+
+try:
+    print(f"Fetching {url}...")
+    response = requests.get(url, timeout=30, allow_redirects=False)
+    print(f"Status Code: {response.status_code}")
+    print(f"Headers: {response.headers}")
+    if response.status_code in [301, 302, 307, 308]:
+        print(f"Redirect Location: {response.headers.get('Location')}")
+        
+except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)